test/update-coveralls

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+#
+# Send coverage data to coveralls.io
+#
+# `go test -coverprofile=` only supports generating a coverage profile for one
+# package at a time. This script combines coverage data from multiple packages
+# and then sends the coverage data to coveralls.io
+#
+echo "mode: set" > acc.out
+FAIL=0
+
+packages="$(find . -type d -maxdepth 3 \
+	! -path './.git*' \
+	! -path '*/_*' \
+	! -path './Godeps*' \
+	! -path './test*' \
```

Comment:
Ah, nice spot. I am thinking maybe we should separate the running of tests from the coverage report. What do you think? It will add ~60 seconds to the build time for Travis, but it keeps the concerns separate.